CHANGELOG.md

```diff
@@ -32,6 +32,7 @@ to [sourceforge feature requests](https://sourceforge.net/p/jabref/features/) by
 
 ### Removed
 - Removed possibility to export entries/databases to an `.sql` file, as the logic cannot easily use the correct escape logic
+- Removed support of old groups format (used prior to JabRef version 1.6)
```

Comment:
Could you clarify that we can still read the old format, hence, automatically convert it to the new group format?